saleor/graphql/shop/mutations.py

```diff
@@ -390,7 +397,12 @@ class Arguments:
         )
 
     class Meta:
-        description = "Update shop order settings."
+        description = (
+            "Update shop order settings across all channels"
+            "\nReturns `orderSettings` for the first `channel` "
```

Comment:
We don't need new-line characters in the descriptions, they usually go into one line unless you want to write some really specific and longer descriptions.